<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AFRWidget</class>
 <widget class="QWidget" name="AFRWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Add new record</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
      <item row="0" column="0">
       <widget class="QLabel" name="label_batch">
        <property name="text">
         <string>Batch</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_qty">
        <property name="text">
         <string>Quantity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
        <item>
         <widget class="QtyInput" name="edit_qty">
          <property name="placeholderText">
           <string>quantity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_max">
          <property name="text">
           <string>max:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_maxvalue">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="combo_batch">
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="action_add">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="action_clear">
       <property name="text">
        <string>C&amp;lear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtyInput</class>
   <extends>QLineEdit</extends>
   <header>QtyInput.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>combo_batch</tabstop>
  <tabstop>edit_qty</tabstop>
  <tabstop>action_add</tabstop>
  <tabstop>action_clear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
